pegasus/helpers/section_api_helpers.rb

```diff
@@ -581,6 +581,9 @@ def self.update_if_owner(params)
       # explicitly clear script_id (unless we're also passed in a valid script id
       # as a param
       fields[:script_id] = nil
+    else
+      # If no course_id provided, make sure we clear an existing course_id
```

Comment:
This should be "If no valid course_id provided..."? Alternately, I'd expect the `else` to be an `elsif`.